src/main/java/io/gravitee/policy/xslt/XSLTTransformationPolicy.java

```diff
@@ -114,6 +114,11 @@ public ReadWriteStream onRequestContent(Request request, ExecutionContext execut
                     xsltTransformationPolicyConfiguration.getParameters().forEach(
                             parameter -> {
                                 if (parameter.getName() != null && ! parameter.getName().trim().isEmpty()) {
+                                    // Apply SpEL conversion
+                                    Object extValue = (parameter.getValue() != null) ? executionContext.getTemplateEngine().getValue(parameter.getValue(), Object.class) : null;
```

Comment:
Why would you get an `Object` instead of a `String` ?

```suggestion
                                    Object extValue = (parameter.getValue() != null) ? executionContext.getTemplateEngine().getValue(parameter.getValue(), String.class) : null;
```

Reply:
You are wright.
I updated the code and fixed the commit message.
I used `git commit --amend` and `git push --force-with-lease` to update the repo.
Is this sufficient?